Integrations/integration-RsaNetWitnessPacketsAndLogs.yml

```diff
@@ -63,32 +63,33 @@ script:
         return Object.keys(obj).length === 0;
     }
 
-    var mapTypeToValue = function (fields) {
-        var result = {};
-        var TYPE = 'type';
+    var fillgroupObject = function (fields, groupObject) {
+        var TYPE  = 'type';
         var VALUE = 'value';
+        var GROUP = 'group';
         if(fields && Array.isArray(fields)) {
             fields.forEach(function (element) {
-                if (element[TYPE] && element[VALUE]) {
-                    result[element[TYPE]] = element[VALUE];
+                if (element[TYPE] && element[VALUE] && element[GROUP]) {
```

Comment:
In case that there is no group key you will not add the element. But I think it will be better if the code will handle those cases if group not exists